src/patternfly/components/Select/select-toggle-typeahead.hbs

```diff
@@ -1,13 +1,15 @@
 {{#if select--IsDisabled}}
   {{#> form-control controlType="input"
     input="true"
+    form-control--IsDisabled='true'
     form-control--modifier="pf-v5-c-select__toggle-typeahead"
-    form-control--attribute=(concat 'type="text" id="' select--id '-typeahead" aria-label="Type to filter" placeholder="' select-typeahead--Placeholder '" disabled')}}
+    form-control--attribute=(concat 'type="text" id="' select--id '-typeahead" aria-label="Type to filter" placeholder="' select-typeahead--Placeholder '"')}}
   {{/form-control}}
 {{else}}
   {{#if select--IsInvalid}}
     {{#> form-control controlType="input"
       input="true"
+      form-control--IsError='true'
       form-control--modifier="pf-v5-c-select__toggle-typeahead"
       form-control--attribute=(concat 'type="text" id="' select--id '-typeahead" aria-invalid="true" value="Invalid" aria-label="Type to filter" placeholder="' select-typeahead--Placeholder '"')}}
     {{/form-control}}
```

Comment:
Duplicate error icons are being rendered in the Invalid example for Select typeahead with this addition since select-toggle is setting an icon as well.